src/main/java/org/folio/circulation/resources/RenewalValidator.java

```diff
@@ -26,6 +26,7 @@
     "Item's loan policy has fixed profile but renewal period is specified";
 
   public static final String DECLARED_LOST_ITEM_RENEWED_ERROR = "item is Declared lost";
+  public static final String CLAIMED_RETURNED_RENEWED_ERROR = "item is Claimed returned";
```

Comment:
Whilst this capitalisation is expected for the status in the UI, I wonder if that makes sense within a sentence.

Please could these error messages be checked with Emma, the Product Owner.

Reply:
Confirmed by Emma

Reply:
Ok, thanks